x-pack/plugins/security_solution/public/management/pages/trusted_apps/state/trusted_apps_list_page_state.ts

```diff
@@ -18,10 +20,34 @@ export interface TrustedAppsListData {
   paginationInfo: PaginationInfo;
 }
 
+/** Store State when an API request has been sent to create a new trusted app entry */
+export interface TrustedAppCreatePending {
```

Comment:
I wonder if there's a way to group these states similar to the List resource.  I know it's not entirely the same since we're creating a resource as opposed to loading it.